{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "964d1191_e75038f5",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1000102
      },
      "writtenOn": "2023-01-12T10:40:35Z",
      "side": 1,
      "message": "Unrelated failures",
      "revId": "76a91d8769b7fff1a4cc784c77b4e19d7716351e",
      "serverId": "8f6f209b-db1a-4cbf-aa44-c8bc30e9bfda"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "89ad75eb_64100ee1",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1000056
      },
      "writtenOn": "2023-01-12T15:29:18Z",
      "side": 1,
      "message": "some nits.",
      "revId": "76a91d8769b7fff1a4cc784c77b4e19d7716351e",
      "serverId": "8f6f209b-db1a-4cbf-aa44-c8bc30e9bfda"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "ca3df765_e89c152c",
        "filename": "bl31/aarch64/runtime_exceptions.S",
        "patchSetId": 2
      },
      "lineNbr": 100,
      "author": {
        "id": 1000056
      },
      "writtenOn": "2023-01-12T15:29:18Z",
      "side": 1,
      "message": "\"FEAT_RAS\"",
      "range": {
        "startLine": 100,
        "startChar": 36,
        "endLine": 100,
        "endChar": 39
      },
      "revId": "76a91d8769b7fff1a4cc784c77b4e19d7716351e",
      "serverId": "8f6f209b-db1a-4cbf-aa44-c8bc30e9bfda"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "64a6c4cb_888ab643",
        "filename": "bl31/aarch64/runtime_exceptions.S",
        "patchSetId": 2
      },
      "lineNbr": 113,
      "author": {
        "id": 1000056
      },
      "writtenOn": "2023-01-12T15:29:18Z",
      "side": 1,
      "message": "not needed as this is expected state from this macro",
      "range": {
        "startLine": 113,
        "startChar": 4,
        "endLine": 113,
        "endChar": 70
      },
      "revId": "76a91d8769b7fff1a4cc784c77b4e19d7716351e",
      "serverId": "8f6f209b-db1a-4cbf-aa44-c8bc30e9bfda"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "84eddabb_308beecc",
        "filename": "bl31/aarch64/runtime_exceptions.S",
        "patchSetId": 2
      },
      "lineNbr": 113,
      "author": {
        "id": 1000102
      },
      "writtenOn": "2023-01-12T15:40:35Z",
      "side": 1,
      "message": "I have removed this in following patch",
      "parentUuid": "64a6c4cb_888ab643",
      "range": {
        "startLine": 113,
        "startChar": 4,
        "endLine": 113,
        "endChar": 70
      },
      "revId": "76a91d8769b7fff1a4cc784c77b4e19d7716351e",
      "serverId": "8f6f209b-db1a-4cbf-aa44-c8bc30e9bfda"
    }
  ]
}